src/appComponents/twitterwall/singletweet.js

```diff
@@ -2,7 +2,7 @@
 angular
     .module('oe.twitterwall')
     .directive('tweet',tweet);
-/*directive to display tweets after a timespan of 10000 ms.*/
+/*directive to display tweets after a timespan of 8000 ms.*/
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected space or tab after "/*" in comment.](https://www.codacy.com/app/triplez/open-event-webapp/file/2412364420/issues/source?bid=3233857&fileBranchId=3235518#l5)